Sources/StreamChat/Query/Filter.swift

```diff
@@ -148,22 +185,45 @@ public extension Filter {
     }
 }
 
+struct FilterAutoKey {
+    let payloadKey: CodingKey
+    let dtoKey: String
+    let value: CVarArg? // isModerator is actually role == admin
```

Comment:
Better to delete this comment and provide better documentation on why we need this 👍